index.js

```diff
@@ -16,11 +16,11 @@ module.exports = function (content) {
       children = result.attributes.children,
       componentProps = renderProps(props);
 
-    let imports = '';
+    let imports = 'import React from "react";';
```

Comment:
Can you be more specific with which issues may occur if React is already defined?

The way I see it is that you can only define it locally by adding it as a dependency in the markdown-file.

```md
---
dependencies:
  React: react
---
```

In that case, I can do a simple check.

If it's globally defined, the import statement will just replace it locally scoped within it's own block. So I can't imagine what could go wrong. Loaders similar to this one doesn't check whether React is defined or not either. E.g: https://github.com/jhamlet/svg-react-loader/blob/master/utility/template.txt